test/src/test/java/org/corfudb/integration/StreamIT.java

```diff
@@ -1,68 +1,53 @@
 package org.corfudb.integration;
 
 import org.corfudb.protocols.wireprotocol.ILogData;
+import org.corfudb.protocols.wireprotocol.LogData;
 import org.corfudb.runtime.CorfuRuntime;
 import org.corfudb.runtime.view.stream.IStreamView;
 import org.junit.Before;
+import org.junit.Test;
 
+import java.util.List;
 import java.util.Random;
 import java.util.UUID;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * A set integration tests that exercise the stream API.
  */
 
-public class StreamIT extends AbstractIT {
```

Comment:
Hm, the surgery here appears to have changed the integration style test into a unit test.  I think it'd be better if this test had adaptations made within the IT style, then, if you wish, add a new unit test(s).

Also, there are two large blocks of code commented out.  If they aren't useful, then perhaps it's better to remove this entire test and/or convert to unit test style only?